### PR TITLE
Make code run successfully in Python3

### PR DESCRIPTION
**Pull request by dpino**
Maybe there is more code that is Python2 dependendant I'm not aware of.

### CODE REVIEW

**Comment by dpino** [comment]
To test these changes, I run the modules that feature unit tests:
  - `./matrixbot/plugins/wkbotsfeeder.py`
  - `./matrixbot/plugins/wktestbotsfeeder.py`
  - `./wkbotsfeeder.py`

File: matrixbot/ldap.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
 
-import ldap as LDAP
+import ldap3 as LDAP
```
Comment (psaavedra) [inline matrixbot/ldap.py:9]: Are you sure ldap3 is a drop-in replacement of ldap? 


Please update the requirements.txt with the corresponding package: `ldap3`

Comment (dpino) [reply matrixbot/ldap.py:9]: I double-checked this change, and you're right it's not necessary. To test it I run the unit-tests and launched an instance of matrix-bot.

In addition to that I update a few other python files that were missing the Python3 shebang.

**Comment by psaavedra** [comment]
maybe @anarute or @pabelenda want to run a review for those changes.

**Review by anarute — APPROVED** [review]
It looks good to me

**Pull request merged**